src/utils/device.js

```diff
@@ -96,3 +96,21 @@ module.exports.isLandscape = function () {
 module.exports.isIOSOlderThan10 = function (userAgent) {
   return /(iphone|ipod|ipad).*os.(7|8|9)/i.test(userAgent);
 };
+
+/**
+ * Check if running in a browser.
+ */
+function isBrowser () {
```

Comment:
I was matching the convention in the file `isIOS ()` and `isGearVR ()` are functions when I don't think they need to be either.